packages/plugin-core/src/_extension.ts

```diff
@@ -978,7 +978,8 @@ async function _setupCommands(
   const existingCommands = await vscode.commands.getCommands();
 
   ALL_COMMANDS.map((Cmd) => {
-    const cmd = new Cmd();
+    const cmd = new Cmd(ws);
```

Comment:
Commands are getting extension passed to their constructor to be able to use an instance of dependency injected extension interface rather than reaching out statically.

Reply:
A command with a no-arg constructor still works fine here right?

Reply:
Yep tested note lookup it has no-arg constructor and appears to work fine. 

```
NoteLookupCommand
  constructor() {
  ```